### PR TITLE
Update to Swift 5.0

### PR DESCRIPTION
**Pull request by theblixguy**

### CODE REVIEW

**Comment by theblixguy** [comment]
Not sure how these files ended up being in conflict 🤔

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 os:
   - osx
-osx_image: xcode9
+osx_image: xcode10.2
 language: swift
 script:
-- set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit-Example -sdk iphonesimulator11.0 -destination "OS=11.0,name=iPhone 6s" | xcpretty
-- xcodebuild -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit_ExampleUITests -sdk iphonesimulator11.0 -destination "OS=11.0,name=iPhone 6s" test -enableCodeCoverage YES
+- set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit-Example -sdk iphonesimulator12.2 -destination "OS=12.2,name=iPhone Xs" | xcpretty
+- xcodebuild -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit_ExampleUITests -sdk iphonesimulator12.2 -destination "OS=12.2,name=iPhone Xs" test -enableCodeCoverage YES
```
Comment (JackTudor96) [inline .travis.yml]: do we definitely want to use a 12.2 sim here? Not sure whether changing the sim and ios version will impact the ui tests in the project

Comment (theblixguy) [reply .travis.yml]: Hmm I ran the tests locally and there were no issues.

Comment (JackTudor96) [reply .travis.yml]: cool, no worries then!

**Pull request merged**